CSS/CSS.sublime-syntax

```diff
@@ -7,6 +7,14 @@ file_extensions:
   - css.erb
 scope: source.css
 variables:
+  # Many variable names taken directly from https://www.w3.org/TR/CSS22/grammar.html#scanner
+  h: '[0-9a-fA-F]'
+  unicode: '\\{{h}}{1,6}(?:\r\n|[ \t\r\n\f])?'
+  escape: '(?:{{unicode}}|\\[^\r\n\f0-9a-fA-F])'
+  nmstart: '(?:[_a-zA-Z]|[^[:ascii:]]|{{escape}})'
+  nmchar: '(?:[_a-zA-Z0-9-]|[^[:ascii:]]|{{escape}})'
+  ident: '-?{{nmstart}}{{nmchar}}*'
+
```

Comment:
Ok, so we can use character set negations (http://www.rexegg.com/regex-class-operations.html#subtraction_javaruby) and unions (http://www.rexegg.com/regex-class-operations.html#union) here. This should result in faster scanning since it doesn't need to branch as with the `|` operator:

```
nmstart: [[_a-zA-Z][\p{L}\p{M}\p{S}\p{N}&&[^[:ascii:]]]]
nmchar: [[-_a-zA-Z0-9][\p{L}\p{M}\p{S}\p{N}&&[^[:ascii:]]]]
```

These are a union of (the listed ASCII characters) and (the unicode characters with those properties, minus all ASCII). Thus effectively limiting matches of the unicode properties to non-ASCII codepoints. This prevents us from accidentally matching any of those ASCII symbols you listed. I specifically 

I suppose we still have an escape branch, but at least we aren't matching junk unicode like control characters, and line and paragraphic breaks.